yt/yt/server/scheduler/scheduler.cpp

```diff
@@ -797,9 +798,11 @@ class TScheduler::TImpl
 
         DoSuspendOperation(
             operation,
-            TError("Suspend operation by user request"),
+            reason.empty()
+                ? TError("Suspend operation by user request")
+                : TError("Suspend operation by user request (Reason: %Qv)", reason),
```

Comment:
We do not use `(Foo: %v)` style formatting in errors, they are for logs only. To add some field to the error, please use smth like:
```
TError error("Suspend operation by user request")
if (!reason.empty()) {
    error <<= TErrorAttribute("reason", reason);
}
```

This will add `reason` as an attribute to the error.